reactioncommerce/reaction-next-starterkit
# Feature: integrate mini cart

Impact: minor
Type: feature

## Summary
Integrates `MiniCart` component that will be rendered on hover over the cart icon. Further, clicking on the cart icon will result in the user being taken to the cart page.

## Testing
1. Hover over cart icon and verify mini cart is rendered
2. Click on cart icon and verify user is taken to the cart page.
